test/suite/test_txn19.py

```diff
@@ -303,7 +303,7 @@ def test_corrupt_log(self):
 
         if expect_fail:
             self.check_file_contains_one_of(errfile,
-            ['/log file.*corrupted/', 'WT_ERROR: non-specific WiredTiger error'])
```

Comment:
It's quite possible now we can change this to `self.check_files_contains('WT_DATA_CORRUPTION: database corruption detected')` since we now have a single error code that we should always get for these errors.